mobile/Ihc/components/MedicationInventory.js

```diff
@@ -108,9 +108,11 @@ export default class MedicationInventory extends Component<{}> {
         />
 
         <Button style={styles.buttonContainer}
-          onPress={() => this.openAddModal()}
+          onPress={this.openAddModal}
           text='Add Medication' />
 
+        <Text style={styles.title}>Medication Inventory{"\n"}</Text>
```

Comment:
why is the {"\n"} necessary? If you are just trying to move the next part to the following line, then try to do it with the style instead

Reply:
Changes from Brent's accident